code/modules/spells/targeted/shoesnatch.dm

```diff
@@ -0,0 +1,32 @@
+/spell/targeted/shoesnatch
+
+	name = "Shoe Snatching Charm"
+	desc = "This spell will allow you to steal somebodies shoes right off of their feet!"
+	school = "evocation"
+	charge_type = Sp_RECHARGE
+	charge_max = 150
+	charge_counter = 0
```

Comment:
charge_counter. It is set to max when the spell is generated anyways, so setting it to 0 in the definition is pointless.